webcompat/webhooks/helpers.py

```diff
@@ -12,6 +12,8 @@
 from webcompat import app
 from webcompat.helpers import proxy_request
 
+MEDIA_PATH = '/tmp/media_issues.txt'
```

Comment:
I guess we probably want this in `config/__init__.py` or `secrets.py.example` at some point.